front/app/containers/Admin/reporting/components/ReportBuilder/Templates/ProjectTemplate.tsx

```diff
@@ -106,11 +124,17 @@ const ProjectTemplate = ({ reportId, projectId }: Props) => {
         {...projectPeriod}
       />
 
-      {participationMethod === 'native_survey' && (
-        <SurveyResultsWidget
-          title={toMultiloc(WIDGET_TITLES.SurveyResultsWidget)}
-          phaseId={phaseId}
-        />
+      {filteredSurveyFields && (
+        <div>
+          {filteredSurveyFields.map((field) => (
```

Comment:
Replace old survey results widget by the single survey question widget (one for each supported question)